server/src/main/java/com/adobe/testing/s3mock/FileStoreController.java

```diff
@@ -470,6 +475,8 @@ public CopyObjectResult copyObject(@PathVariable final String destinationBucket,
       produces = "application/x-www-form-urlencoded")
   public void getObject(@PathVariable final String bucketName,
       @RequestHeader(value = RANGE, required = false) final Range range,
+      @RequestHeader(value = IFMATCH, required = false) final List<String> match,
```

Comment:
please use std. Headers from `org.springframework.http.HttpHeaders`. `BetterHeaders` is meant to wrap AWS API specific headers.
Same applies to `IFNONEMATCH` Header